src/components/nodes-table/AssociationGroups.vue

```diff
@@ -197,7 +211,7 @@ export default {
     getSourceAddress () {
       return {
         nodeId: this.group.node.id,
-        endpoint: this.group.nodeEndpoint || 0
+        endpoint: this.group.group.endpoint || 0
```

Comment:
I know we talked about zwave-js not caring about missing endpoint vs endpoint 0. However, that was for value IDs. Associations are different, because endpoint 0 gets treated as a Multi Channel Association, while missing endpoint gets treated as a "classic" node Association.

Reply:
Yeah my question was about this 😆 No problem BTW now I fix it :)